### ISSUE TITLE
Duplicates when drag and drop events in yearview

### ISSUE THREAD

**Issue by hoksilato** [tpruvot/fullcalendar#11]
When I using drag and drop events to yearview calendar, it's duplicated 12 times (as 12 month).

P/S: I used drag and drop of AdminLTE template at https://almsaeedstudio.com/themes/AdminLTE/pages/calendar.html

**Comment by tpruvot** [comment]
Indeed, seems to be a new problem i can reproduce, it was ok in the past...
